package.json

```diff
@@ -9,7 +9,7 @@
   },
   "dependencies": {
     "babel-eslint": "4.1.3",
-    "eslint": "codeclimate/eslint.git#9dba1fe",
+    "eslint": "codeclimate/eslint.git#d24d0b451b75cfad304faeb394371096f2c90777",
```

Comment:
Is there a reason for preferring the shortened sha? I went the other way in a recent csslint bump with the following reasoning: This makes for easier editing (git log or the GitHub UI most often gives you full shas).